scripts/zkai_stwo_query_grinding_budget_gate.py

```
@@ -0,0 +1,555 @@
+        "claims_new_frontier": claims_new_frontier,
+    }
+
+
+def policies(inventory: list[dict[str, Any]]) -> list[dict[str, Any]]:
+    return [
+        policy(
+            inventory,
+            "fixed_layout_budget_1",
+            ("fixed_adjacent_layout",),
+            "BASELINE_NO_GRINDING",
+            False,
```

Ruff (0.15.13)

[warning] Boolean positional value in function call

(FBT003)